app/server/sonicpi/test/test_tick.rb

```diff
@@ -118,6 +118,15 @@ def test_keyed_ticks
       assert_equal(0, tick(:baz))
     end
 
+    def test_string_keyed_ticks
+      tick_reset_all
+      assert_equal(0, tick("foo"))
```

Comment:
Shouldn't it be like this:

assert_equal(1, tick("foo"))

Or I misunderstood what's going on?